### PR TITLE
libcomps: init at 0.1.8

### PR DESCRIPTION
**Pull request by FRidh**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
- I will open a PR upstream for the patch
- Currently I have the following issue:

```
$ nix-shell -p 'python3.withPackages(ps: with ps; [ libcomps ])' -I nixpkgs=. --run "python -c 'import libcomps'"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/nix/store/j5dgh01vxx4jq4in1i7fc2c30hirbjml-python3-3.6.3-env/lib/python3.6/site-packages/libcomps/__init__.py", line 1, in <module>
    from ._libpycomps import *
ImportError: libz.so.1: cannot open shared object file: No such file or directory
```

File: pkgs/development/libraries/libcomps/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  sourceRoot = "${src.name}/${pname}";
+
+  cmakeFlags="-DPYTHON_DESIRED=${stdenv.lib.substring 0 1 python.pythonVersion}";
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
+  buildInputs = [ libxml2 expat python check ];
+
+  meta = with stdenv.lib; {
+    description = "Libcomps is alternative for yum.comps library";
```
Comment (williammpratt) [inline pkgs/development/libraries/libcomps/default.nix]: This description doesn't say what it does. It just says that it does the same as something else. => It is not a description.

**Comment by FRidh** [comment]
> Why can't upstream accept your patch?

I did not mention they could/would not accept the patch.

**Comment by FRidh** [comment]
Don't need this anymore so closing.

**Pull request closed without merging**